cloudinit/config/cc_vyos.py

```diff
@@ -250,6 +253,37 @@ def set_ipaddress(config, iface_type: str, iface: str, address: str,
         config.set_tag(['interfaces', iface_type, iface, 'vif'])
 
 
+# configure MTU for Ethernet
+def set_ether_mtu(config, iface: str, mtu: int) -> None:
+    """Configure MTU for Ethernet interface
+
+    Args:
+        config (_type_): configuration object
+        iface (str): interface name
+        mtu (int): MTU value
+    """
+    logger.debug("Setting MTU for {}: {}".format(iface, mtu))
+    # get maximum possible MTU
+    iplink = run(['ip', '-json', '-detail', 'link', 'show', 'dev', iface],
+                 capture_output=True)
+    if iplink.returncode != 0:
+        logger.debug("Cannot get interface details for {}".format(iface))
```

Comment:
Does it use python2? Why old `format`

Reply:
It was kept for compatibility with Python < 3.6. Now this is not the problem anymore, but I think it is better to convert all messages and reformat the whole file at once than mix old/new styles.